src/NinjaTools.FluentMockServer/Xunit/MockServerContainer.cs

```diff
@@ -1,30 +1,37 @@
 using System;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
 using System.Linq;
 using System.Net.Http;
 using System.Net.NetworkInformation;
 using System.Threading.Tasks;
 using Ductus.FluentDocker.Builders;
 using Ductus.FluentDocker.Services;
 
+
 namespace NinjaTools.FluentMockServer.Xunit
 {
     /// <summary>
     /// The InMemory handle to the MockServer Docker Container
     /// </summary>
     public class MockServerContainer : IDisposable
     {
+        public const string Prefix = "NinjaTools-FluentMockServer";
+        public const string Suffix = "netcore-xunit-background-runner";
+        public static string ContainerName = $"{Prefix}-{Suffix}";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4885942)